### PR TITLE
fix export of output files zip, because of bad usage of base path par…

### PR DESCRIPTION
**Pull request by schwitzguebel**
…ameter for minio upload

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: docker-compose/core-valid/config/core-valid-gridcapa-task-manager-application.yml [hunk 1]
```diff
@@ -65,10 +65,11 @@ task-server:
       - REMEDIAL-ACTIONS-RESULT
       - REX-RESULT
     enable-export-logs: false
+    manual-upload-base-path: CORE/VALID
 
 minio-adapter:
   bucket: ${GRIDCAPA_MINIO_BUCKET:gridcapa}
-  base-path: ${GRIDCAPA_MINIO_BASE_PATH:/CORE/VALID}
+  base-path: ${GRIDCAPA_MINIO_BASE_PATH:CORE/VALID}
```
Comment (vbochetRTE) [inline docker-compose/core-valid/config/core-valid-gridcapa-task-manager-application.yml:72]: Shouldn't the basepath be empty by default if no environment variable is found?

File: k8s/bases/core-cc/core-cc-task-manager-configmap.yaml [hunk 2]
```diff
@@ -85,12 +85,13 @@ data:
           - CGM_OUT
           - METADATA
         enable-export-logs: false
+        manual-upload-base-path: CORE/CC
       whitelist:
         - http://gridcapa-minio:9000/
 
     minio-adapter:
       bucket: gridcapa
-      base-path: ${MINIO_BASE_DIRECTORY}
```
Comment (vbochetRTE) [inline k8s/bases/core-cc/core-cc-task-manager-configmap.yaml:93]: Did you forget to remove the `MINIO_BASE_DIRECTORY` variable from the `core-cc-task-manager-deployment`?

File: k8s/bases/core-cc/core-cc-task-manager-configmap.yaml [hunk 3]
```diff
@@ -85,12 +85,13 @@ data:
           - CGM_OUT
           - METADATA
         enable-export-logs: false
+        manual-upload-base-path: CORE/CC
```
Comment (vbochetRTE) [inline k8s/bases/core-cc/core-cc-task-manager-configmap.yaml:88]: Not sure if this value should be hardcoded in the configmap. I would have preferred having them added as an environment variable in `env.properties` file and loaded via deployment file.

(applies for all processes in `k8s` directory)

File: k8s/bases/core-cc/core-cc-task-manager-configmap.yaml (hunk 2 above)
Comment (schwitzguebel) [reply k8s/bases/core-cc/core-cc-task-manager-configmap.yaml:93]: no in fact that is how it was set beforer .... MINIO_BASE_DIRECTORY isn't set in docker local configs

File: docker-compose/core-valid/config/core-valid-gridcapa-task-manager-application.yml (hunk 1 above)
Comment (schwitzguebel) [reply docker-compose/core-valid/config/core-valid-gridcapa-task-manager-application.yml:72]: in docker config the value isn't set ... but you are right could be removed

**Pull request merged**